core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -1980,7 +1980,7 @@ oppia.constant('STATE_ERROR_MESSAGES', {
   UNABLE_TO_END_EXPLORATION: (
     'There\'s no way to complete the exploration starting from this card.'),
   INCORRECT_SOLUTION: (
-    'This solution does not lead to another state.')
+    'Current solution does not lead to another card.')
```

Comment:
Ditto. You might want to pull out this string into a constant, btw; we seem to use it multiple times.